lib/LgTvController.js

```diff
@@ -126,6 +126,7 @@ class LgTvController extends EventEmitter {
     this.tvOn = false;
     this.mediaPaused = false; // internal property to keep track if media is paused on tv (only works if the home app remote control is used)
     this.isScreenOn = true; // internal property to keep track if the screen is turned off or on, is there an event from the tv which can tell me that instead?
+    this.pictureMode = ''; // internal property to keep track of the active picture mode. There is no way at the moment to retrieve the active picture mode from the TV
```

Comment:
I would preferer to have the `null` as initial.

Reply:
Done